### DISCUSSION TITLE
Getting "//" as one of the packages in the output of --dry-run=json

### DISCUSSION THREAD

**Discussion by aserafin** [vercel/turborepo#4255] [Help]
When running the `turbo build` command with filter based on github commit I'm getting the `//` as one of the entries in packages array - is this expected behaviour/documented somewhere or I'm doing sth wrong? 🤔 

```
npx -y turbo build --dry-run=json --filter=...[commit-sha]
```

Result:

```
"packages": [
    "//",
    "first-app",
    "second-app",
    "utils"
  ]
```

**Comment by mehulkar** [comment]
You aren't doing anything wrong. The `//` sigil means "root". I'm guessing that you have a `//#build` task in your turbo.json. We could translate this to a prettier name if you open up an issue!
